src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/CohortDomainServiceTests.cs

```diff
@@ -121,6 +124,24 @@ public async Task OverlapOnEndDate_Validation()
             _fixture.VerifyOverlapExceptionOnEndDate();
         }
 
+        [TestCase(Party.Provider)]
+        [TestCase(Party.Employer)]
+        public async Task UpdateDraftApprenticeship_IsSuccessful_ThenDraftApprenticeshipIsUpdatedAndLastUpdatedFieldsAreSet(Party withParty)
+        {
+            _fixture.WithParty(withParty).WithExistingCohort(withParty).WithExistingDraftApprenticeship();
+            await _fixture.UpdateDraftApprenticeship();
+            _fixture.VerifyDraftApprenticeshipUpdated();
+            _fixture.VerifyLastUpdatedFieldsAreSet(withParty);
+        }
+
+        [Test]
+        public async Task UpdateDraftApprenticeship_IsSuccessful_ThenLastUpdatedFieldsAreNotSet()
```

Comment:
Could this be renamed to include why the last updated fields aren't set? e.g. `UpdateDraftApprenticeship_WhenUserInfoDoesNotExist_ThenDraftApprenticeshipLastUpdatedFieldsAreNotSet`